backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidApplicationConfiguration.java

```diff
@@ -37,6 +37,9 @@
 
 	/** whether to use the accelerometer. default: true **/
 	public boolean useAccelerometer = true;
+	
+	/** whether to use the gyroscope. default: true **/
+	public boolean useGyroscope = true;
```

Comment:
I'm finding issue with this.  Unfortunately, I think it's a deeper issue.  I set useGyroscope = false in GDX source, exported GDX to my test environment, and my test stopped working.  Regardless if I enabled it in my AndroidLauncer.java in onCreate()
```java
		AndroidApplicationConfiguration config;
		
		config = new AndroidApplicationConfiguration();
		config.useGyroscope = true; 
		config.useAccelerometer = true;
		config.useCompass = true;
		initialize(app, config);
		
		super.onCreate(savedInstanceState);
```
with this configuration, Gdx.input.isPeripheralAvailable(Peripheral.Gyroscope) always returns false, if it's set to false in GDX source.

to determine if there is something I was missing in my implementation, I set useAccelerometer and useCompass GDX defaults to false.  With the same (above) config, now Gdx.input.isPeripheralAvailable(Peripheral.Accelerometer) also never comes back true.  

In addition, I did the reverse..... set GDX defaults to true across the board, and my app config all to false.  Both
Gdx.input.isPeripheralAvailable(Peripheral.Accelerometer) and Gdx.input.isPeripheralAvailable(Peripheral.Gyroscope) return true, disregarding the config parameters at initialize().  

Looking.....